README.md

```diff
@@ -33,40 +38,63 @@ mongodb = "0.10.0"
 bson = "0.14.1"
 ```
 
+#### Configuring the async runtime
+The driver supports both of the most popular async runtime crates, namely [`tokio`](https://crates.io/crates/tokio) and [`async-std`](https://crates.io/crates/async-std). By default, the driver will use [`tokio`](https://crates.io/crates/tokio), but you can explicitly choose a runtime by specifying one of `"tokio-runtime"` or `"async-std-runtime"` feature flags in your `Cargo.toml`.
+
+For example, to instruct the driver to work with [`async-std`](https://crates.io/crates/async-std), add the following to your `Cargo.toml`:
+```toml
+[dependencies.mongodb]
+version = "0.10.0"
+default-features = false
+features = ["async-std-runtime"]
+```
+
+#### Enabling the sync API
+The driver also provides a blocking sync API. To enable this, add the `"sync"` feature to your `Cargo.toml`:
+```toml
+[dependencies.mongodb]
+version = "0.10.0"
+default-features = false
+features = ["sync"]
+```
+**Note:** if the sync API is enabled, the async-specific types will be privatized (e.g. `mongodb::Client`). The sync-specific types can be imported from `mongodb::sync` (e.g. `mongodb::sync::Client`).
+
 ## Example Usage
 Below are simple examples of using the driver. For more specific examples and the API reference, see the driver's [docs.rs page](https://docs.rs/mongodb).
-### Connecting to a MongoDB deployment
+
+### Using the async API
+#### Connecting to a MongoDB deployment
 ```rust
 use mongodb::{Client, options::ClientOptions};
 ```
 ```rust
 // Parse a connection string into an options struct.
-let mut client_options = ClientOptions::parse("mongodb://localhost:27017")?;
+let mut client_options = ClientOptions::parse("mongodb://localhost:27017").await?;
 
 // Manually set an option.
 client_options.app_name = Some("My App".to_string());
 
 // Get a handle to the deployment.
-let client = Client::with_options(client_options)?;
+let client = Client::with_options(client_options).await?;
```

Comment:
I think this constructor doesn't return a future?

Reply:
fixed